StreetSmartAPI/StreetSmart.WPF.Tests/StreetSmartAPITests.cs

```diff
@@ -131,4 +132,133 @@ public void CloseViewer_Empty()
     Mock.Verify();
   }
 #endif
-}
+  [Fact]
+  public async Task AddOverlay_ValidJson_ReturnsOverlay()
+  {
+    var geoJsonOverlay = new GeoJsonOverlay("{\"type\":\"FeatureCollection\",\"features\":[]}", "myName", "mySrs", Color.AliceBlue);
+    _browserMock.Setup(x => x.IsDisposed).Returns(false);
+    _browserMock.Setup(x => x.GetBrowser()).Returns(_iBrowserMock.Object);
+    _browserMock.Setup(x => x.ExecuteScriptAsync(It.IsAny<string>())).Callback(() => _viewer.OnResult(new { id = 4 }, "AddOverlay1")).Verifiable(Times.Once);
```

Comment:
Is it OK to have It.IsAny string as input parameter? Do we need to validate input parameter format here?